tests/templates/test_embeddings/test_basis.py

```diff
@@ -76,7 +76,20 @@ def circuit2():
 class TestInputs:
     """Test inputs and pre-processing."""
 
-    @pytest.mark.parametrize("x", [[0], [0, 1, 1]])
+    @pytest.mark.parametrize(
+        "feat,wires,expected",
```

Comment:
```suggestion
        ("feat","wires","expected"),
```

This should fix the coverage problems